doc/modules/model_evaluation.rst

```diff
@@ -1033,6 +1033,60 @@ set [0,1] has an error: ::
     for an example of zero one loss usage to perform recursive feature
     elimination with cross-validation.
 
+.. _brier_score_loss:
+
+Brier score loss
+----------------
+
+The :func:`brier_score_loss` function computes the
+`Brier score <https://en.wikipedia.org/wiki/Brier_score>`_
+for binary classes. Quoting Wikipedia:
+
+   "The Brier score is a proper score function that measures the accuracy of
+    probabilistic predictions. It is applicable to tasks in which predictions
+    must assign probabilities to a set of mutually exclusive discrete outcomes.
+    The set of possible outcomes can be either binary or categorical in nature,
```

Comment:
We only support the binary case in our implementation right? I would not mention the categorical case if we don't support it.